### DISCUSSION TITLE
Github wants a pull request that is not necessary

### DISCUSSION THREAD

**Discussion by TreasureMaster** [community/community#32530] [Pull Requests]
Hello! I have a local repository with a few branches. I was working in a local repo but now I need to post the code. I use github desktop and publish the first branch and then the second. But github offers to merge these branches - "compare & pull request". Why? I don't want to merge. I need these branches now with different directions. What can be done to prevent this?

**Comment by airtower-luna** [comment]
You don't have to merge them, you can just ignore the suggestion if that's what you want. :slightly_smiling_face: 

Making branches to merge them later is just the common usage, so it's often convenient to be able to initiate a PR with just one click.

**Comment by TreasureMaster** [comment]
Thank you. I found the answer. Those green and red buttons are distracting, which is why I didn't notice the white merge cancel button.

**Comment by jfullstackdev** [comment]
yup that's the correct idea for branches, for example in a certain company or open source, many will work on a certain feature but it does not mean it will be pushed to main, because main is already for production basically, so no buggy feature should ever slip unchecked 

it's also, as for me, a copy of your recent progress even if it's not yet complete and you want to make sure, it will not be lost, so you will not repeat the same job again when the local copy is lost
